decidim-accountability/app/controllers/decidim/accountability/admin/results_controller.rb

```diff
@@ -8,9 +8,15 @@ class ResultsController < Admin::ApplicationController
         include Decidim::ApplicationHelper
         include Decidim::SanitizeHelper
         include Decidim::Proposals::Admin::Picker
+        include Decidim::Accountability::Admin::Filterable
 
         helper_method :results, :parent_result, :parent_results, :statuses, :present
 
+        def collection
+          parent_id = params[:parent_id].presence
+          @collection ||= Result.where(component: current_component, parent_id: parent_id).page(params[:page]).per(15)
```

Comment:
@jinjagit the description says the user can change the amount of results per page (15, 50 100), but the number is hard-coded here to 15?

Did you forget about this? Or should the description be updated?

Reply:
I believe this means it will, initially, display 15 results per page, but that the user can then select 50, 100 or return to 15 per page, and this can be seen in the screen-cast gif in this PR.   

I will experiment with removing the hard-coded limit, to see if this improves the user experience, but I think the description is correct in either case (but could perhaps be more detailed).

Reply:
@jinjagit sorry, you're absolutely right! 

As a side note, GIFs are a little hard to deal with, but now GitHub allows uploading videos, which are easier to control from the viewer side 😄